src/OSPSuite.TeXReporting/Builder/TextTeXBuilder.cs

```diff
@@ -6,22 +6,22 @@
 
 namespace OSPSuite.TeXReporting.Builder
 {
-   internal class TextTeXBuilder : TeXChunkBuilder<Text>
+   public abstract class TextTeXBuilder<T> : TeXChunkBuilder<T> where T : Text
```

Comment:
I had to abstract that class with T so that I could simply override the TeXChunk method